docs/dev/existing_data_updates.rst

```diff
@@ -288,6 +302,45 @@ common errors will occur when new CEMS plants lack timezone data in the EIA data
 See section 6.B.1 for instructions on how to fix this. Once you've updated the
 spreadsheet tracking these errors, reload the ``epacems`` assets in Dagster.
 
+D. NREL ATB
+^^^^^^^^^^^^
+**4.D.1)** Materialize the ``_core_nrelatb__ transform_start`` asset in Dagster. If
+there are new primary keys or ``core_metric_parameters``, this should raise errors. New
+core parameters should be renamed in ``core_metric_parameters_rename``, and new primary
+keys should be renamed in ``rename_dict``. Debug any remaining errors.
+
+**4.D.2)** If there are any new primary columns (e.g.,
+``model_tax_credit_case_nrelatb``), add them to the ``idx`` of the table whose
+``core_metric_parameters`` they describe as a primary key. You may have to create a new
+table, as needed.
```

Comment:
What is the difference between a primary column and a primary key?

Reply:
No difference, reworded to clarify.